### PR TITLE
Splunk hub fixes

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Proposed Changes

This PR fixes typos in the plugin description and the help.md

## Testing: 
NA - Doc only change.

### CODE REVIEW

File: splunk/help.md [hunk 1]
```diff
@@ -23,7 +23,20 @@
 
 To connect to Splunk, you must have valid credentials and network access to the Splunk API port (Splunk's default is TCP/8089). This plugin supports both the Free and Enterprise Splunk licenses.
 
-If use the Splunk free license, configure your Splunk instance to allow remote login by adding the following line to the general stanza in `$SPLUNK_HOME/etc/system/local/server.conf`:
+To configure an alert trigger, the Splunk server must be able to connect back to the Komand server via http(s) ports to send webhook events.
```
Comment (jschipp-r7) [inline splunk/help.md]: We’ll likely want to indicate that this is specific to Komand customer and if they want to do the same thing use InsightConnect app for Splunk with a link to it, that way we cover both bases.

Comment (jschipp-r7) [reply splunk/help.md]: Looks like the trigger section was removed in help.md too in this PR

Comment (jmcadams-r7) [reply splunk/help.md]: Manually added trigger. Change the comment.

File: splunk/help.md [hunk 2]
```diff
@@ -220,27 +220,7 @@ This action is used to return the job history of a specified saved search.
...
-|names|[]string|None|True|Names of the alerts to hook|None|
-
-##### Output
-
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|results_link|string|False|None|
-|result|object|False|None|
-|search_name|string|False|None|
+_This plugin does not contain any triggers._
```
Comment (lmilby-r7) [inline splunk/help.md]: Alerts get removed?

File: splunk/tests/alert.json [hunk 3]
```diff
@@ -1,16 +1,5 @@
 {
     "body": {
-        "connection": {
```
Comment (lmilby-r7) [inline splunk/tests/alert.json]: Add this back to tests

File: splunk/help.md (hunk 2 above)
Comment (gusingh-r7) [reply splunk/help.md]: There is no trigger in plugin.spec.yaml

File: splunk/tests/alert.json (hunk 3 above)
Comment (jschipp-r7) [reply splunk/tests/alert.json]: Yep, this file is useless without the schema

File: splunk/help.md [hunk 4]
```diff
@@ -23,7 +25,22 @@
 
 To connect to Splunk, you must have valid credentials and network access to the Splunk API port (Splunk's default is TCP/8089). This plugin supports both the Free and Enterprise Splunk licenses.
 
-If use the Splunk free license, configure your Splunk instance to allow remote login by adding the following line to the general stanza in `$SPLUNK_HOME/etc/system/local/server.conf`:
+To configure an alert trigger, the Splunk server must be able to connect back to the InsightConnect server via http(s) ports to send webhook events.
```
Comment (jschipp-r7) [inline splunk/help.md]: Remove this, since trigger is no longer in this plugin

File: splunk/help.md [hunk 5]
```diff
@@ -23,7 +25,22 @@
...
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|credentials|credential_username_password|None|False|Username and password|None|
+|host|string|None|True|Hostname or IP address of Splunk server to connect to e.g. splunk.example.com|None|
+|license|string|None|True|License type for Splunk host|['Enterprise', 'Free']|
+|port|integer|8089|True|Port the Splunk API is listening on. Default is 8089|None|
+|ssl_verify|boolean|None|True|Verify server's SSL/TLS certificate|None|
+|use_ssl|boolean|None|True|Whether or not to use SSL|None|
+
+To configure an alert trigger, the Splunk server must be able to connect back to the InsightConnect server via http(s) ports to send webhook events.
```
Comment (jschipp-r7) [inline splunk/help.md]: Remove this duplicate line of 21

File: splunk/help.md [hunk 6]
```diff
@@ -23,7 +25,22 @@
...
+
+To configure your Splunk instance to allow remote login by adding the following line to the general stanza in `$SPLUNK_HOME/etc/system/local/server.conf`:
```
Comment (mrinehart-r7) [inline splunk/help.md]:
```suggestion
To configure your Splunk instance to allow remote login, add the following line to the general stanza in `$SPLUNK_HOME/etc/system/local/server.conf`:
```

File: splunk/help.md [hunk 7]
```diff
@@ -34,40 +51,44 @@ allowRemoteLogin = always
 
 There's no authentication in the free license, so set `license` to `Free` and omit input to the username and password fields.
 
-To configure an alert trigger, the Splunk server must be able to connect back to the Komand server via http(s) ports to send webhook events.
+To get Splunk alerts or send saved searches to InsightConnect, please use the [InsightConnect Splunk App](https://splunkbase.splunk.com/app/4673/).
```
Comment (mrinehart-r7) [inline splunk/help.md]: Duplicate paragraph, remove.

File: splunk/help.md (hunk 6 above)
Comment (mrinehart-r7) [reply splunk/help.md]: Missing the line that needs to be added, as per instruction

File: splunk/help.md (hunk 2 above)
Comment (jmcadams-r7) [reply splunk/help.md]: Yea, this was removed in 3.0.0...it's Komand specific.

**Pull request merged**